app/src/main/kotlin/com/wire/android/ui/authentication/login/Login.kt

```diff
@@ -67,7 +75,11 @@ private fun LoginContent() {
                 Spacer(modifier = Modifier.height(16.dp))
                 ForgotPasswordLabel(modifier = Modifier.fillMaxWidth())
             }
-            LoginButton(modifier = Modifier.fillMaxWidth(), email = email.text, password = password.text)
+            LoginButton(modifier = Modifier.fillMaxWidth(), email = email.text, password = password.text) {
```

Comment:
I think those field should rather consume a String field coming from the ViewModel State object